src/gen-use-table.py

```diff
@@ -293,6 +293,9 @@ def map_to_use(data):
 
 		# Resolve Indic_Syllabic_Category
 
+		# TODO: There are Tibetan signs that don't have UISC
+		if U in [0xF18, 0xF19, 0xF3E, 0xF3F, 0xF86, 0xf87]: UISC = Cantillation_Mark
```

Comment:
Always minimum four digits for Unicode hex values. Also, last one has lowercase 'f' while others are uppercase.

Reply:
Sure, can you help on finding correct syllable category for those also?

Reply:
Let's ask @jfkthame @PeterCon 
Typically those need to be discussed on an issue by themselves.

Reply:
@dscorbett

Reply:
@PeterCon Peter, can you double check the overrides for Tibetan against the list posted on the spec page? Better yet, can you refresh the override list for all scripts?